### ISSUE TITLE
folder with .js in name

### ISSUE THREAD

**Issue by kevinya** [stejskal/sbt-webpack#3]
Hi,

I tried to use your webpack plugin.
However it doesn't work with dependencies having ".js" in their folder name.
Could you add a check for folder with ".js" in the name ?
Thanks.

Here is the dist log:
`
[info] Done packaging.
[info] 
[info] 
java.io.FileNotFoundException: /home/kevin/Documents/workspace/pit/java/pit/target/web/web-modules/main/webjars/lib/zone.js (Is a directory)
    at java.io.FileInputStream.open(Native Method)
    at java.io.FileInputStream.<init>(FileInputStream.java:138)
    at sbt.Using$$anonfun$fileInputStream$1.apply(Using.scala:73)
    at sbt.Using$$anonfun$fileInputStream$1.apply(Using.scala:73)
    at sbt.Using$$anon$2.openImpl(Using.scala:65)
    at sbt.OpenFile$class.open(Using.scala:43)
    at sbt.Using$$anon$2.open(Using.scala:64)
    at sbt.Using$$anon$2.open(Using.scala:64)
    at sbt.Using.apply(Using.scala:23)
    at sbt.Hash$.apply(Hash.scala:55)
    at sbt.RichFile.hash(Path.scala:46)
    at stejskal.sbt.WebpackPlugin$$anonfun$runWebpack$1$$anonfun$apply$3$$anonfun$2.apply(WebpackPlugin.scala:55)
    at stejskal.sbt.WebpackPlugin$$anonfun$runWebpack$1$$anonfun$apply$3$$anonfun$2.apply(WebpackPlugin.scala:53)
    at scala.collection.IndexedSeqOptimized$class.foldl(IndexedSeqOptimized.scala:51)
    at scala.collection.IndexedSeqOptimized$class.foldLeft(IndexedSeqOptimized.scala:60)
    at scala.collection.mutable.ArrayBuffer.foldLeft(ArrayBuffer.scala:47)
    at stejskal.sbt.WebpackPlugin$$anonfun$runWebpack$1$$anonfun$apply$3.apply(WebpackPlugin.scala:53)
    at stejskal.sbt.WebpackPlugin$$anonfun$runWebpack$1$$anonfun$apply$3.apply(WebpackPlugin.scala:47)
    at scala.Function$$anonfun$chain$1$$anonfun$apply$1.apply(Function.scala:24)
    at scala.Function$$anonfun$chain$1$$anonfun$apply$1.apply(Function.scala:24)
    at scala.collection.LinearSeqOptimized$class.foldLeft(LinearSeqOptimized.scala:111)
    at scala.collection.immutable.List.foldLeft(List.scala:84)
    at scala.collection.TraversableOnce$class.$div$colon(TraversableOnce.scala:138)
    at scala.collection.AbstractTraversable.$div$colon(Traversable.scala:105)
    at scala.Function$$anonfun$chain$1.apply(Function.scala:24)
    at com.typesafe.sbt.web.SbtWeb$$anonfun$projectSettings$43.apply(SbtWeb.scala:215)
    at com.typesafe.sbt.web.SbtWeb$$anonfun$projectSettings$43.apply(SbtWeb.scala:215)
    at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
    at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
    at sbt.std.Transform$$anon$4.work(System.scala:63)
    at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
    at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
    at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
    at sbt.Execute.work(Execute.scala:235)
    at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
    at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
    at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
    at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)
[error] (*:webPipeline) java.io.FileNotFoundException: /home/kevin/Documents/workspace/pit/java/pit/target/web/web-modules/main/webjars/lib/zone.js (Is a directory)
[error] Total time: 4 s, completed Mar 21, 2016 10:28:44 AM
`

`
build.sbt

libraryDependencies ++= Seq(
        // Typescript
        "org.webjars.npm" % "typescript" % "1.8.2",

```
    // Angular2
    "org.webjars.npm" % "angular2" % "2.0.0-beta.9",

    // Import Dynamique
    "org.webjars.npm" % "systemjs" % "0.19.20",

    // Stockage de données partagés pour les tâches asynchrones
    "org.webjars.npm" % "zone.js" % "0.5.15",

    // Décorateur
    "org.webjars.npm" % "reflect-metadata" % "0.1.3",

    // Nécessaire à IE
    "org.webjars.npm" % "es6-promise" % "3.0.2",
    "org.webjars.npm" % "es6-shim" % "0.34.1",
```

)

// Webpack
pipelineStages := Seq(webpack)
includeFilter in webpack := "index.js"
webpackConfig in webpack := Some(file("webpack.config.js"))
`

**Comment by stejskal** [comment]
Thanks for the feedback, I added a quick check to ensure files being added were normal files which i believe will fix your problem.  Can you do a build from master and test it?

**Comment by kevinya** [comment]
It does work. Thanks !

**Comment by stejskal** [comment]
great!  i published a new version, 0.4, with the fix.
